src/pykeen/models/mocks.py

```diff
@@ -7,28 +7,32 @@
 
 import torch
 
-from .base import EntityRelationEmbeddingModel, Model
+from .base import Model
 from ..triples import CoreTriplesFactory
 
 __all__ = [
     "MockModel",
 ]
 
 
-class MockModel(EntityRelationEmbeddingModel):
+class MockModel(Model):
     """A mock model returning fake scores."""
 
     hpo_default = {}
 
-    def __init__(self, *, triples_factory: CoreTriplesFactory, **_kwargs):
+    def __init__(self, *, triples_factory: CoreTriplesFactory, embedding_dim: int, **_kwargs):
```

Comment:
`embedding_dim` is not used?

Reply:
It's not, but it's necessary for the CLI not to break :/ I guess I can just turn those tests off. Your call

Reply:
Fixed this